web/public/index.html

```diff
@@ -67,7 +67,25 @@ <h3><i class="material-icons">chat_bubble_outline</i> Friendly Chat</h3>
         <button hidden id="sign-out" class="mdl-button mdl-js-button mdl-js-ripple-effect mdl-color-text--white">
           Sign-out
         </button>
-        <button hidden id="sign-in" class="mdl-button mdl-js-button mdl-js-ripple-effect mdl-color-text--white">
+
+        <form hidden id="sign-in-form" action="#">
```

Comment:
Q: Why changing to email/password? Isn't Google sign-in easier for adding MFA later since it marks email as verified automatically?

Reply:
That's a fair point - I mainly did this b/c I figured that we'd need the email/password sign in for the current blocking function example and thought it might be clearer to just focus on one sign in and build off that vs having multiple sign in's. I wonder if we keep the google sign in (easier to work with anyway) and maybe do a blocking function example that's like "filter out email addresses that start with letter ___"? WDYT? Don't want to step on your toes either since this is your presentation

Reply:
Discussed offline, going with Google provider for the example and users can have Google accounts that end in different domains aside from `@gmail.com`